src/utils/appearance.ts

```diff
@@ -1,16 +1,69 @@
+import { getAudience } from 'types/component-props/_mixins';
 import { ContentProps, ContentType } from 'types/content-props/_content-common';
+import { getContentLanguages } from './languages';
 
 const contentTypeWithWhiteBackground = [
     ContentType.CurrentTopicPage,
-    ContentType.MainArticle,
     ContentType.FormIntermediateStepPage,
     ContentType.FormsOverview,
+    ContentType.MainArticle,
 ];
 
-export const hasWhiteBackground = (content: ContentProps) => {
+const contentTypesWithWhiteHeader: ReadonlySet<ContentType> = new Set([
+    ContentType.AreaPage,
+    ContentType.CurrentTopicPage,
+    ContentType.FormIntermediateStepPage,
+    ContentType.FormsOverview,
+    ContentType.FrontPage,
+    ContentType.FrontPageNested,
+    ContentType.GenericPage,
+    ContentType.GuidePage,
+    ContentType.OfficeBranchPage,
+    ContentType.OfficeEditorialPage,
+    ContentType.Overview,
+    ContentType.PressLandingPage,
+    ContentType.ProductPage,
+    ContentType.SituationPage,
+    ContentType.ThemedArticlePage,
+]);
+
+export const hasNoTopGap = (content: ContentProps) => {
+    if (
+        content.type === ContentType.FrontPage ||
+        content.type === ContentType.FrontPageNested
+    ) {
+        return (
+            getAudience(content.data.audience) === 'employer' ||
+            getAudience(content.data.audience) === 'provider'
+        );
+    }
+
+    return false;
+};
+
+export const hasWhiteHeader = (content: ContentProps) => {
+    const { type } = content;
+
+    if (type === ContentType.MainArticle) {
+        const contentType = content.data?.contentType;
+        return contentType === 'news' || contentType === 'pressRelease';
+    }
+
+    return contentTypesWithWhiteHeader.has(type);
+};
+
+export const hasWhitePage = (content: ContentProps) => {
     if (content.type === ContentType.MainArticle) {
         return content.data.contentType !== 'lastingContent';
     }
```

Comment:
Disse to er egentlig samme sjekken uttrykt på forskjellige måter, kanskje det bør være en felles funksjon?

Reply:
Ikke sikker på om vi kan ha en felles funksjon, fordi en side kan ha hvit header, men være grå ellers? Men jeg har skrevet om litt slik at det blir mindre repeterende logikk.